File: sdkwork-app-sdk-typescript/src/types/blocked-user-vo.ts
/** 黑名单用户响应 */
export interface BlockedUserVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 用户ID */
  userId?: string;
  /** 用户名 */
  username?: string;
  /** 昵称 */
  nickname?: string;
  /** 头像 */
  avatar?: string;
  /** 拉黑时间 */
  blockTime?: string;
  /** 拉黑原因 */
  blockReason?: string;
}

File: sdkwork-app-sdk-csharp/Models/BlockedUserVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class BlockedUserVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? UserId { get; set; }
        public string? Username { get; set; }
        public string? Nickname { get; set; }
        public string? Avatar { get; set; }
        public string? BlockTime { get; set; }
        public string? BlockReason { get; set; }
    }
}
